util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941160.yaml

```diff
@@ -1,6 +1,7 @@
 ---
   meta: 
     author: "csanders-git"
+    author: "Franziska Bühler"
```

Comment:
FTR, while this is still valid in yaml, it will replace the original author.

Reply:
Yeah, probably better to comma separate the names like 942120 and friends.

Reply:
Oh, I didn't know that. I am on vacation and I have no laptop with me. Could you please push into my branch? I hope, that is possible...
Or you merge and I will fix that in a seperate PR afterwards.